examples/Atlas/urdf/atlas_convex_hull.urdf

```diff
@@ -1,998 +1,1138 @@
 <?xml version="1.0" ?>
-<!-- =================================================================================== -->
-<!-- |    This document was autogenerated by xacro from atlas_full_contact_no_hands.urdf.xacro | -->
-<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
-<!-- =================================================================================== -->
 <robot name="atlas" xmlns:xacro="http://www.ros.org/wiki/xacro">
   <link name="l_clav">
     <inertial>
-      <mass value="3.45"/>
-      <origin rpy="0 -0 0" xyz="0 0.048 0.084"/>
-      <inertia ixx="0.011" ixy="0" ixz="0" iyy="0.009" iyz="-0.004" izz="0.004"/>
-    </inertial>
-    <visual>
-      <origin rpy="0 -0 0" xyz="0 0 0"/>
-      <geometry>
-        <mesh filename="package://Atlas/urdf/meshes/l_clav.wrl" scale="1 1 1"/>
-      </geometry>
-    </visual>
-    <collision>
-      <origin rpy="0 -0 0" xyz="0 0 0"/>
-      <geometry>
-        <mesh filename="package://Atlas/urdf/meshes/l_clav_chull.wrl" scale="1 1 1"/>
-      </geometry>
-    </collision>
-  </link>
-  <link name="l_farm">
-    <inertial>
-      <mass value="3.388"/>
-      <origin rpy="0 -0 0" xyz="0 -0.065 0"/>
-      <inertia ixx="0.00656" ixy="0" ixz="0" iyy="0.00358" iyz="0" izz="0.00656"/>
+      <mass value="4.466"/>
+      <origin rpy="0 0 0" xyz="0 0 -0.084"/>
+      <inertia ixx="0.011" ixy="0" ixz="0" iyy="0.009" iyz="0.004" izz="0.004"/>
     </inertial>
     <visual>
-      <origin rpy="0 -0 0" xyz="0 0 0"/>
       <geometry>
-        <mesh filename="package://Atlas/urdf/meshes/l_farm.wrl" scale="1 1 1"/>
+        <mesh filename="package://Atlas/urdf/meshes/r_clav.obj"/>
```

Comment:
This is due to the fixes we did for the left arm. The upper arm links on the left arm arm in fact the same as the ones on the right arm, some just have a pi rotation offset, so they should point to the same mesh file. We could delete the unused meshes for the left arm.